cartoframes/examples.py

```diff
@@ -0,0 +1,5 @@
+from cartoframes import CartoContext
+
+def exampleContext():
```

Comment:
How about just `Examples` like you proposed in https://github.com/CartoDB/cartoframes/issues/170#issuecomment-364934921? I feel the user doesn't need to specifically know this is another `CartoContext` (which for us is an implementation detail).

Reply:
Not sure. If the user doesn't know that it's a `CartoContext`, how can he or she know what API can be used? I don't think that documentation should duplicate the `CartoContext` one.

Whatever the question is, uppercasing the first letter is pythonic?

Both questions are driven by my Python ignorance, you know 😅 I'm asking just to understand the best way to do this, I'll do what you think is better :-)